regparser/tree/xml_parser/appendices.py

```diff
@@ -270,6 +278,8 @@ def is_subhead(tag, text):
             elif initial_marker(text) and child.tag in ('P', 'FP', 'HD'):
                 self.paragraph_with_marker(
                     text, tree_utils.get_node_text_tags_preserved(child))
+            elif child.tag in ('P', 'FP') and child.get('SOURCE') == 'FP-DASH':
+                self.paragraph_with_dash(text)
```

Comment:
Hmm, I see what you're saying (side note: note sure why I squashed this in between marker/no-marker—no matter what I'll change that, simply for clarity). I basically saw this as a "paragraph without a marker that fills with dashes", but you're right, it is orthogonal to that decision-making.